README.md

```diff
@@ -1,216 +1,265 @@
-<img src="readme_images/android_app_running.png" height="200px">
+   <img src="readme_images/run_on_android.png" height="60px">
+   
+   <img src="readme_images/android_app_running.png" height="200px">
 
 <details>
-  <summary>Using Gradle</summary>
+  <summary>Alternatively, use Gradle</summary>
 
-`./gradlew installDebug` - install Android application on an Android device (on a real device or on an emulator)
+To install an Android application on a real device Android device or an emulator, run `./gradlew installDebug` in the terminal.
 
 </details>
 
-## iOS
+### On iOS
 
-We suggest going through the "Hello, World" steps of creating and deploying a sample project in Xcode to a simulator and/or your physical device.
-A video tutorial for setting up Xcode and running your first "Hello, World" application is available in [this Standford CS193P lecture recording](https://youtu.be/bqu6BquVi2M?start=716&end=1399).
+#### Running on a simulator
 
-### Running on an iOS simulator
+To run your application on an iOS simulator in Android Studio, modify the `iosApp` run configuration:
 
-Once you have configured your environment correctly, you will be able to select which iOS simulator to run your application in Android Studio on by modifying the `iosApp` run configuration.
+1. In the list of run configurations, select **Edit Configurations**:
 
-Select "Run" | "Edit Configurations..." and navigate to the "iOS Application" | "iosApp" run configuration. In the "Execution target" drop-down, select your target device.
+   <img src="readme_images/edit_run_config.png" height="200px">
 
-<img src="readme_images/edit_run_config.png" height="200px">
-<img src="readme_images/target_device.png">
-
-Press the "Run" button to run your Compose Multiplatform app on the iOS simulator.
-
-<img src="readme_images/hello_world_ios.png" height="200px">
+2. Navigate to **iOS Application** | **iosApp**.
+3. In the **Execution target** list, select your target device. Click **OK**:
 
-### Running on a real iOS device
+   <img src="readme_images/target_device.png">
 
-Running your Compose Multiplatform application on a physical device can be done for free. You need:
-- an [Apple ID](https://support.apple.com/en-us/HT204316)
-- the registered iOS device in Xcode
+4. The `iosApp` run configuration is now available. Click **Run** next to your virtual device:
 
-Before you continue, make sure that you can successfully run a plain "Hello, World" application from Xcode on your physical device.
-
-To run the application, set the `TEAM_ID` associated with your Apple ID in `iosApp/Configuration/Config.xcconfig`.
+<img src="readme_images/hello_world_ios.png" height="200px">
 
-#### Finding your Team ID
-Use `kdoctor --team-ids` to find and set your Team ID. This will list all Team IDs currently configured on your system, for example:
-```
+#### Running on a real device
+
+You can run your Compose Multiplatform application on a real iOS device.
```

Comment:
Let's point out that it's possible for free. Apples Developer membership programs can be confusing :)